2022q4/wifi.adoc

```diff
@@ -0,0 +1,34 @@
+=== Wireless updates
+
+Links: +
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/\~bz/wireless/[https://people.freebsd.org/~bz/wireless/] +
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+During the 4th quarter of 2022 not much work was publicly visible and admittedly slightly slow.
+Behind the scenes wireless work was happening on two fronts:
+
+ * work on 11n and 11ac and wpa bits,
+
+ * and continous work on more drivers and firmware, problems, testing, and filling more gaps for these.
+
+While the main development for newer standards and the Intel iwlwifi driver is sponsored by the FreeBSD Foundation, I find myself spending a lot of private time with Realtek rtw88 and rtw89, Atheros, and Mediatek mt76 (7921 and 7915) drivers now as well.
+Testing changes on bare metal and using bhyve VMs using passthru has become more time-consuming, with the amount of supported chipsets increasing, in order not to break other drivers.
+Even different (generations of) chipsets supported by the same driver, at times, behave differently with the same change.
+
+A separate discussion was brought to me about the size of firmware added to the tree for the already existing drivers and the firmware for more drivers coming.
+The chicken-egg problem to solve is to have the firmware available on the release media as otherwise a lot of modern laptops will not have any sort of outside communications available at the time of install in their default configuration should we not commit it to main anymore.
```

Comment:
> The chicken-egg problem to solve is to have the firmware available on the release media as otherwise a lot of modern laptops will not have any sort of outside communications available at the time of install in their default configuration should we not commit it to main anymore.

That's a relatively long sentence, with no commas. 

Also, it's ambiguous; as if including firmware _is_ the problem.

Reply:
I don't see ambiguity here, but that may be only me.
```suggestion
The chicken-egg problem to solve is to have the firmware available on the release media; without it, a lot of modern laptops will not have any sort of outside communications available at the time of install in their default configuration.
```

Reply:
A _solution_ to a problem will be to do something. In this case, what's done is: 

* having firmware.

I see: "the _problem_ is to have firmware".

Reply:
Which is correct, because of the limited space on distribution media. @bzfbd, is that what you meant?